index.html

```diff
@@ -0,0 +1,2204 @@
+                              <p class="small">required</p>
+                            </td>
+                            <td>The patient’s state.</td>
+                          </tr>
+                          <tr>
+                            <td>
+                              <p class="cell"><code class="attribute">zip</code> <code>string</code></p>
+                              <p class="small">required</p>
+                            </td>
+                            <td>The patient’s zip code.</td>
+                          </tr>
+                          <tr>
+                            <td>
+                              <p class="cell"><code class="attribute">country</code> <code>string</code></p>
+                              <p class="small">required</p>
+                            </td>
+                            <td>The patient’s country.</td>
+                          </tr>
+                          <tr>
+                            <td>
+                              <p class="cell"><code class="attribute">phone</code> <code>string</code></p>
+                            </td>
+                            <td>The address's phone number.</td>
+                          </tr>
+                          <tr>
+                            <td>
+                              <p class="cell"><code class="attribute">latitude</code> <code>number</code></p>
+                              <p class="small">required</p>
+                            </td>
+                            <td>The latitude of the address for map purposes.</td>
+                          </tr>
+                          <tr>
+                            <td>
+                              <p class="cell"><code class="attribute">longitude</code> <code>number</code></p>
+                              <p class="small">required</p>
+                            </td>
+                            <td>The longitude of the address for map purposes.</td>
+                          </tr>
+                          <tr>
+                            <td>
+                              <p class="cell"><code class="attribute">timezone</code> <code>string</code></p>
+                              <p class="small">required</p>
+                            </td>
+                            <td>The address's timezone.</td>
+                          </tr>
+                        </tbody>
+                      </table>
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+            <div class="col-6">
+              <h5>Example Request JSON</h5>
+              <pre>
+                <code>
+  {
+    "first_name": "Joseph",
+    "last_name": "Schmoseph",
+    "dob": "1977-06-01",
+    "gender": "female",
+    "phone": "+17183211234",
+    "email": "testguy@example.com",
+    "photo_url": "http://example.com/pic.png",
```

Comment:
```suggestion
    "photo_url": "https://example.com/pic.png",
```
I know it's for display only, but for details sake.